### DISCUSSION TITLE
What format do I need to prepare data for processing.

### DISCUSSION THREAD

**Discussion by ZGHHGZ** [isce-framework/dolphin#109] [Q&A]
### Checked for duplicates

Yes - I've already checked

### Alternatives considered

Yes - and alternatives don't suffice

### Related problems

What format do I need to prepare data for processing. can you give an example？thank you very much.

### Describe the feature request

I need or want [...]

**Comment by scottstanie** [comment]
Hi, the software uses GDAL for most of its image reading, so any coregistered SLC format that is GDAL-compatible should work. Most testing is on the outputs of [COMPASS](https://github.com/opera-adt/COMPASS) (which uses HDF5), but I have used it for the standard isce2 outputs for flat binary files too.

I will probably be finishing up #14 in the coming weeks to help more people test things out.
